joomla/joomla-cms
# Move default logs directory

Pull Request for Issue #7611
#### Summary of Changes

Moves the application's default logs directory to `administrator/logs`, changes include:
- Updated .gitignore
- Updated the build script to only include the default directory in the full (new) install package; updates and patches won't get this
- The sysinfo screen uses the new path as the default lookup for the log path
- The install app uses the new path as the default setting
#### B/C implications

I ain't got a clue to be honest.  Since Joomla isn't installable as an extension of itself anymore, the issues I raised previously regarding that on the linked issue shouldn't be a problem.  I guess the worst thing is if someone's got a hardcoded `JPATH_ROOT . '/logs'` log path in their scripts.
#### Testing Instructions

On a new install of Joomla, the default log path should be `administrator/logs`.

When updating to a Joomla version including this package, the `administrator/logs` directory should NOT be added to your filesystem.

The `log_path` configuration parameter is NOT changed at update, a site owner can/should apply this change at their discretion once accepted.
